spec.html

```diff
@@ -36640,10 +36633,8 @@
                   1. Let _fenvRec_ be _fenv_'s EnvironmentRecord.
                   1. Let _benv_ be the running execution context's LexicalEnvironment.
                   1. Let _benvRec_ be _benv_'s EnvironmentRecord.
-                  1. Let _fobj_ be _benvRec_.GetBindingValue(_F_, *false*).
-                  1. Assert: _fobj_ is never an abrupt completion.
-                  1. Let _status_ be _fenvRec_.SetMutableBinding(_F_, _fobj_, *false*).
-                  1. Assert: _status_ is never an abrupt completion.
+                  1. Let _fobj_ be ! _benvRec_.GetBindingValue(_F_, *false*).
+                  1. Let _status_ be ! _fenvRec_.SetMutableBinding(_F_, _fobj_, *false*).
```

Comment:
This `_status_` is never used, should it be `Perform ! _fenvRec_.SetMutableBinding(_F_, _fobj_, *false*).`?